core/capabilities/fakes/consensus_nodag.go

```diff
@@ -71,14 +71,17 @@ func (fc *fakeConsensusNoDAG) close() error {
 // NOTE: This fake capability currently bounces back the request payload, ignoring everything else.
 // When the real NoDAG consensus OCR plugin is ready, it should be used here, similarly to how the V1 fake works.
 func (fc *fakeConsensusNoDAG) Simple(ctx context.Context, metadata capabilities.RequestMetadata, input *sdkpb.SimpleConsensusInputs) (*valuespb.Value, error) {
-	fc.eng.Infow("Executing Fake Consensus NoDAG", "input", input)
+	fc.eng.Errorw("Executing Fake Consensus NoDAG", "input", input)
 
 	switch obs := input.Observation.(type) {
 	case *sdkpb.SimpleConsensusInputs_Value:
 		if obs.Value == nil {
 			return nil, errors.New("input value cannot be nil")
 		}
-		return obs.Value, nil
+		return valuespb.NewMapValue(map[string]*valuespb.Value{
+			"payload":  obs.Value,
+			"metadata": valuespb.NewMapValue(map[string]*valuespb.Value{}),
```

Comment:
That's not what we were planning to do. Simple() strips away metadata and signatures.
Why do you want to have metadata here? If you want to send something onchain, you need to use outputs of Report().

Please also always add @ettec as reviewer.

Reply:
There shouldn't be any "payload" key in the returned value here. @MStreet3 removed it in https://github.com/smartcontractkit/cre-sdk-go/pull/13/files

Maybe a dep bump is needed somewhere.